packages/main/src/Dialog.js

```diff
@@ -98,6 +98,10 @@ const metadata = {
 		 * If this property is set to true, the Dialog will be draggable by its header.
 		 * <br><br>
 		 * <b>Note:</b> The component can be draggable only in desktop mode.
+		 * <br><br>
+		 * <b>Note:</b> This property overrides the default HTML "draggable" attribute native behavior.
+		 * When "draggable" is set to true, the native browser "draggable"
```

Comment:
I suggest to clarify that this is about the property and not the attribute::
* When  the "draggable" property is set to true, the native browser "draggable"